apps/masterbots.ai/components/routes/chat/chat-message.tsx

```
@@ -52,34 +54,43 @@ export function ChatMessage({
                   </ClickableText>
                 </li>
               )
             },
+            //* Custom code block component with syntax highlighting
             code({ node, inline, className, children, ...props }) {
+              //* Handle cursor animation for streaming responses
               if (children.length) {
-                if (children[0] == '▍') {
-                  return <span className="mt-1 cursor-default animate-pulse">▍</span>
-                }
+                if (children[0] == '▍')
```

Biome

[error] Use === instead of ==. == is only allowed when comparing against `null`

== is only allowed when comparing against null

Using == may be unsafe if you are relying on type coercion
Unsafe fix: Use ===


(lint/suspicious/noDoubleEquals)